packages/react-core/src/components/CalendarMonth/CalendarMonth.tsx

```diff
@@ -20,6 +20,15 @@ export enum Weekday {
   Saturday
 }
 
+export interface InlineCalendarProps {
+  /** Component wrapping the calendar month when used inline. Recommended to be 'article'. */
+  wrapperComponent?: keyof JSX.IntrinsicElements;
+  /** Title of the calendar rendered above the inline calendar month. Recommended to be a 'title' component. */
+  title?: React.ReactNode;
+  /** Id of the accessible label of the calendar month. Recommended to map to the title. */
+  ariaLabeledby?: string;
```

Comment:
@thatblindgeye If the consumer actually passes `InProps`. should we require the `ariaLabelledby` prop?

Reply:
The only downside to making it required is that if the `wrapperComponent` inlineProp is passed "div", the ARIA labeling attributes aren't well supported, so `aria-labelledby` shouldn't be passed in in that case.